### PR TITLE
Got objects/nodes to 100% and added small coverage to other

### PR DESCRIPTION
**Pull request by mlanotte**
Added small addition to other file

### CODE REVIEW

File: tendrl/commons/tests/objects/alert/test_alert_init.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
+import __builtin__
+import maps
+from mock import patch
+from tendrl.commons import objects
```
Comment (anmolsachan) [inline tendrl/commons/tests/objects/alert/test_alert_init.py]: Try keeping tendrl imports separate from library imports. We can have a line in between them.

**Comment by mlanotte** [comment]
I added the line in between the imports

**Comment by mlanotte** [comment]
Just realized me and Nathan both did the alert_init__ so I'm not sure how we should go about it. Both of our files make the coverage as high as possible

**Comment by mlanotte** [comment]
Is there a way for my  tendrl/commons/tests/objects/alert/test_alert_init.py to be added but not Nathan's?
He worked on it first and I didn't realize it but his is only giving the file 94% coverage while mine is 100% so it would be better to take mine instead.

**Comment by nathan-weinberg** [comment]
I will remove mine from my PR :(

**Pull request merged**